### PR TITLE
Updated Deprecated and Fixed Broken Links

### PR DESCRIPTION
**Pull request by Sayanta66**
- Updated the links
- Fixed the broken links

### CODE REVIEW

File: source/deploy/consider-mobile-vpn-options.rst
Comment (justinegeffen) [inline source/deploy/consider-mobile-vpn-options.rst:11]: This link has a 301 redirect and is already going to the correct page. :)

File: source/deploy/consider-mobile-vpn-options.rst
Comment (justinegeffen) [inline source/deploy/consider-mobile-vpn-options.rst:23]:
```suggestion
We also recommend following our `recommended steps to secure your deployment <https://docs.mattermost.com/deploy/consider-mobile-vpn-options.html>`__ and to review the following commonly-asked questions about data security on mobile devices:
```

Comment (justinegeffen) [reply source/deploy/consider-mobile-vpn-options.rst:23]: It's always best to have the URL without the appended search terms. :)

File: source/deploy/consider-mobile-vpn-options.rst
Comment (justinegeffen) [inline source/deploy/consider-mobile-vpn-options.rst:51]: This link is repeated - is it relevant to both points?

Comment (justinegeffen) [reply source/deploy/consider-mobile-vpn-options.rst:51]: @cwarnermm, these four links didn't have 301s and I'm not sure where they're meant to point (but the dev docs don't seem to be a perfect fit).

**Review by justinegeffen — COMMENTED** [review]
Thanks for this PR. I have some questions so I have looped @cwarnermm in. :) Great job on this!

File: source/deploy/consider-mobile-vpn-options.rst
Comment (Sayanta66) [reply source/deploy/consider-mobile-vpn-options.rst:23]: Okay, This is a valuable addition. Thanks a lot.

File: source/deploy/consider-mobile-vpn-options.rst
Comment (Sayanta66) [reply source/deploy/consider-mobile-vpn-options.rst:11]: Can you help me understand what is 301 redirect? @justinegeffen . It will help me in future.

Comment (justinegeffen) [reply source/deploy/consider-mobile-vpn-options.rst:11]: It's a way to make sure older pages go to the new ones automatically. It's not part of the process that you need to follow though! :) And we have discussed this change and it works for us. :)

Comment (Sayanta66) [reply source/deploy/consider-mobile-vpn-options.rst:11]: Okay okay thanks a lot

File: source/deploy/consider-mobile-vpn-options.rst
Comment (cwarnermm) [reply source/deploy/consider-mobile-vpn-options.rst:51]: Thanks @Sayanta66 and @justinegeffen!

This Docathon 2021 contribution has revealed that previously-existing content is missing from the current doc set. I've re-added this content via https://github.com/mattermost/docs/pull/4871

Once this new PR is merged, the 5 links above in @Sayanta66's PR should be updated as follows:
- deploy/mobile-faq.html#how-do-i-receive-mobile-push-notification-if-my-it-policy-requires-the-use-of-a-corporate-proxy-server
- deploy/mobile-faq.html#deploy-mattermost-in-a-proxy-aware-configuration-with-a-pre-proxy-relay
- deploy/mobile-faq.html#deploy-mattermost-with-connection-restricted-post-proxy-relay-in-dmz-or-a-trusted-cloud-environment
- deploy/mobile-faq.html#whitelist-mattermost-push-notification-proxy-to-bypass-your-corporate-proxy-server
deploy/mobile-faq.html#run-app-store-versions-of-the-mattermost-mobile-apps

**Comment by Sayanta66** [comment]
@justinegeffen @cwarnermm I think this PR is now eligible to merge

**Comment by justinegeffen** [comment]
> @justinegeffen @cwarnermm I think this PR is now eligible to merge

Thanks @Sayanta66! Please take a look at the final comment on the PR - once the referenced PR is merged, those links need to be updated and then we can merge this one. :)

**Comment by Sayanta66** [comment]
> > @justinegeffen @cwarnermm I think this PR is now eligible to merge
> 
> Thanks @Sayanta66! Please take a look at the final comment on the PR - once the referenced PR is merged, those links need to be updated and then we can merge this one. :)

Okay, so how will I update this PR?

**Comment by justinegeffen** [comment]
When the referenced PR shows as "Merged" then the links can all be updated with the ones that @cwarnermm listed in her comment. You can update each link individually inline as a comment or by editing the file directly in GitHub. Once the pending PR is merged we'll be more than happy to help with updating this one. :)

**Comment by Sayanta66** [comment]
@justinegeffen okay waiting for the #4871 to get merged

**Review by justinegeffen — APPROVED** [review]

**Comment by justinegeffen** [comment]
Closed as source repo has been deleted and this can't be merged.

**Pull request closed without merging**
